src/reportWebVitals.js

```diff
@@ -1,6 +1,8 @@
-const reportWebVitals = onPerfEntry => {
+const reportWebVitals = (onPerfEntry) => {
   if (onPerfEntry && onPerfEntry instanceof Function) {
-    import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
+    import('web-vitals').then(({
+      getCLS, getFID, getFCP, getLCP, getTTFB,
+    }) => {
```

Comment:
- [ ] Kindly remove unused files that are generated while running `create-react-app` as it makes your code unclean.

Reply:
Ok I will delete all the unnecessary files

Reply:
@Wes-Isaac I deleted the unwanted files